source/includes/stackpath/_sites.md

```diff
@@ -961,3 +961,36 @@ Optional | &nbsp;
 `backupOrigin.username` <br/> *string* | Username to use when authenticating with the backup origin. 
 `backupOrigin.password` <br/> *string* | Password to use when authenticating with the backup origin. 
 `backupOrigin.commonCertificateName` <br/> *string* | Common name to validate SSL origin requests against.
+
+<!-------------------- RETRIEVE EDGESSL SETTINGS -------------------->
+
+### Retrieve EdgeSSL settings
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/api/v1/services/stackpath/test-area/edgesslsettings/eb3ecdbe-d73b-40e6-a263-166accba75ed"
```

Comment:
oh nice, you dont need the siteId to retrieve edgeSSL ?

Reply:
Not as a query parameter, It just needs to be part of the route I believe (the same as origin, waf, and cdn)